hw/ip/adc_ctrl/data/adc_ctrl.hjson

```diff
@@ -164,10 +172,6 @@
         async: "clk_aon_i",
         resval: "0",
         fields: [
-//          { bits: "1:0",
```

Comment:
i never removed this part because the original design wanted to keep these around as reserved fields just in case.

Reply:
oh I see. is there a way to declare these explicitly as "reserved"? 
because right now these are just comments - so these bit locations no not get allocated / interpreted by reggen...

Reply:
screw it, let's just get rid of it. I think the fact that we are not starting from bit0 or whatever is a sign that we are reserving.
I think i was respecting the designer choice at the time, but that doesn't really apply now.

Reply:
OK then - let's remove these comments.